providers/fips/self_test_data.inc

```diff
@@ -1641,6 +1715,26 @@ static const ST_KAT_SIGN st_kat_sign_tests[] = {
         ITM(ecdsa_bin_expected_sig)
     },
 # endif
+# ifndef OPENSSL_NO_ECX
+    {
+        OSSL_SELF_TEST_DESC_SIGN_EDDSA,
+        "ED448",
+        NULL,
+        ed448_key,
+        NULL, 0, NULL, 0, NULL, 0,
```

Comment:
Hi,

When I pick and test this commit, I found it should be as the following, or it will failed on self-tests.

```
        ITM(sig_kat_entropyin),
        ITM(sig_kat_nonce),
        ITM(sig_kat_persstr),
```

Therefore, I would like to know whether it is a typo here, or do I misunderstand something?

Thank you.

Reply:
I will have to reorder the code since this doesnt print anything when it fails here.

Reply:
EDDSA works without the entropy.

Reply:
I found that it failed in `set_kat_drbg`.

Since EDDSA works without the entropy, we may need to skip the KAT setup for it?

Reply:
yes - I need to shift this inside the function so that it runs the self test callback with an error set..